pom.xml

```
@@ -66,7 +66,7 @@
       <dependency>
         <groupId>io.jenkins.tools.bom</groupId>
         <artifactId>bom-2.387.x</artifactId>
-        <version>2357.v1043f8578392</version>
+        <version>2496.vddfca_753db_80</version>
```

2543.vfb_1a_5fb_9496d is the most recent release of the 2.387.x bom line.
```suggestion
        <version>2543.vfb_1a_5fb_9496d</version>
```